presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -59,6 +60,7 @@
     public static final String DISTRIBUTED_INDEX_JOIN = "distributed_index_join";
     public static final String HASH_PARTITION_COUNT = "hash_partition_count";
     public static final String PARTITIONING_PROVIDER_CATALOG = "partitioning_provider_catalog";
+    public static final String MATERIALIZE_EXCHANGES_STRATEGY = "materialize_exchanges_strategy";
```

Comment:
nit: `exchange_materialization_strategy` sounds better to me (it works as an English phrase).